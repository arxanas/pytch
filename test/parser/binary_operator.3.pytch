1 + 2 or 3 + 4 and 5 + 6
