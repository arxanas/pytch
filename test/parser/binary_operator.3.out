SyntaxTree
    BinaryExpr
        BinaryExpr
            IntLiteralExpr
                Token '1'
            Leading ' '
            Token '+'
            IntLiteralExpr
                Leading ' '
                Token '2'
        Leading ' '
        Token 'or'
        BinaryExpr
            BinaryExpr
                IntLiteralExpr
                    Leading ' '
                    Token '3'
                Leading ' '
                Token '+'
                IntLiteralExpr
                    Leading ' '
                    Token '4'
            Leading ' '
            Token 'and'
            BinaryExpr
                IntLiteralExpr
                    Leading ' '
                    Token '5'
                Leading ' '
                Token '+'
                IntLiteralExpr
                    Leading ' '
                    Token '6'
                    Trailing '\n'
    Token EOF ''
